Dockerfile

```diff
@@ -1,19 +1,27 @@
-FROM node:8
+FROM node:10
 
 # Create app directory
 WORKDIR /usr/src/app
 
 # Install app dependencies
 # A wildcard is used to ensure both package.json AND package-lock.json are copied
 # where available (npm@5+)
-COPY package*.json ./
+#COPY package*.json ./
+COPY . .
+
 
+RUN npm set unsafe-perm true
+RUN npm i eslint-plugin-import eslint-plugin-flowtype 
```

Comment:
It would be better if these dependencies were in package.json.

Reply:
I'm a little confused, why do you need to install additional dependencies? Isn't eslint in the node_modules/.bin/ folder? I was able to run eslint by running `npm run eslint` IIRC.

Reply:
I have to agree with @apclary, these additional dependencies shouldn't be required.

Reply:
Also minor thing but we could use a higher docker version, I used 10 for the executables but open-minded if we should use something even higher.

Reply:
You're right. I had to do that previously when I was copying in the app source via `COPY package*.json`. Using `COPY . .` makes the separate install step unnecessary.